launcher/net/Download.cpp

```diff
@@ -116,7 +116,11 @@ void Download::executeTask()
     if (APPLICATION->capabilities() & Application::SupportsFlame
             && request.url().host().contains("api.curseforge.com")) {
         request.setRawHeader("x-api-key", APPLICATION->getFlameAPIKey().toUtf8());
-    };
+    } else if (request.url().host().contains("api.modrinth.com")) {
+        QString token = APPLICATION->getModrinthAPIToken();
+        if (!token.isNull())
```

Comment:
The default value of the setting is `""`, which should be an empty but non-null value. 
```suggestion
        if (!token.isEmpty())
```

Reply:
getModrinthAPIToken returns a null QString if the setting is empty... should this be changed?

Reply:
Oh. Nevermind! I assumed that the function would always just pass the settings value no matter what. This is fine as is.